tests/framework/class-sensei-unsupported-theme-handler-faux-page-imitator.php

```diff
@@ -0,0 +1,87 @@
+<?php
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Test shim for page imitator abstract class.
+ *
+ * @author Automattic
+ *
+ * @since 1.12.0
+ */
+class Sensei_Unsupported_Theme_Handler_Faux_Page_Imitator
```

Comment:
Hmm. Now the comments say they are for the Course Results page, but I'm assuming that this class is generic and can be used for any tests. Should we maybe remove those references to any specific page to avoid confusion down the road?